akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -197,9 +197,11 @@ object MediaType {
     r.get
   }
 
-  sealed abstract class Binary(val value: String, val mainType: String, val subType: String, val comp: Compressibility,
+  sealed abstract class Binary(val value: String, val _mainType: String, val _subType: String, val comp: Compressibility,
```

Comment:
`_mainType` and `_subType` don't have to be `val`.

```suggestion
  sealed abstract class Binary(val value: String, _mainType: String, _subType: String, val comp: Compressibility,
```